api/apps/api/src/modules/scenarios/locks/lock.service.spec.ts

```diff
@@ -0,0 +1,226 @@
+import { Test } from '@nestjs/testing';
+import { getRepositoryToken } from '@nestjs/typeorm';
+import { EntityManager } from 'typeorm';
+import * as faker from 'faker';
+
+import { FixtureType } from '@marxan/utils/tests/fixture-type';
+import {
+  lockedScenario,
+  lockNotFoundError,
+  LockService,
+} from '@marxan-api/modules/scenarios/locks/lock.service';
+import {
+  JobStatus,
+  Scenario,
+  ScenarioType,
+} from '@marxan-api/modules/scenarios/scenario.api.entity';
+import { ScenarioLockEntity } from '@marxan-api/modules/scenarios/locks/scenario.lock.entity';
+import { User } from '@marxan-api/modules/users/user.api.entity';
+import { left, right } from 'fp-ts/Either';
+
+let fixtures: FixtureType<typeof getFixtures>;
+
+describe('Lock service test suite', () => {
```

Comment:
[minor[ whole `describe` is redundant and not giving any value in particular - only increases indentation level.

Reply:
@ikas if you set a watcher in specs, it shall run all tests affected by change - but you are right, that could be handy - so may be matter of preference (running/reading - I still bet on reading as it happens often 🤷 )